rotkehlchen/accounting/export/csv.py

```diff
@@ -289,10 +292,28 @@ def create_zip(
 
         return success, filename
 
-    def to_csv_entry(self, event: 'ProcessedAccountingEvent') -> dict[str, Any]:
+    def to_csv_entry(self, event: Union['ProcessedAccountingEvent', 'HistoryBaseEntry']) -> dict[str, Any]:  # noqa: E501
         """Prepare the provided event to have a common format for the accounting
         CSV exported file.
         """
+        if isinstance(event, HistoryBaseEntry):
+            dict_event = {
+                'identifier': event.identifier,
+                'entry_type': event.entry_type.serialize(),
+                'event_identifier': event.event_identifier,
```

Comment:
We already have the serialize method and this seems to be doing the same. Can't you just call it and then for the balances flattern it manually? meaning pop the value in serialize and add to the mapping the two values required